applications/nrf_desktop/src/modules/dfu_mcumgr.c

```diff
@@ -4,8 +4,15 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include <zephyr/kernel.h>
+
+#if defined CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT
 #include <zephyr/dfu/mcuboot.h>
 #include <zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h>
+#elif defined CONFIG_DESKTOP_DFU_BACKEND_SUIT
```

Comment:
Let's get rid of this #elif as well

Reply:
Can do it, but prefer not to include files which are 100% not used. IMO leaving it as it is much better option.

Reply:
Well, this header is used in the SUIT configuration of this module. It's like `IS_ENABLED` macro, the build system will figure out that this header is not needed in the MCUboot configuration as there is no reference to its APIs

Reply:
I don't insist but let's see what others think

Reply:
If we still need to ifdef the mcuboot part then imo we can leave it as is.